eng/Versions.props

```diff
@@ -146,25 +154,21 @@
     <MicrosoftPrivateIntellisenseVersion>10.0.0-preview-20251006.1</MicrosoftPrivateIntellisenseVersion>
     <!-- MsQuic -->
     <MicrosoftNativeQuicMsQuicSchannelVersion>2.4.16</MicrosoftNativeQuicMsQuicSchannelVersion>
-    <!-- emscripten / Node
-         Note: when the name is updated, make sure to update dependency name in eng/pipelines/common/xplat-setup.yml
-               like - DarcDependenciesChanged.Microsoft_NET_Workload_Emscripten_Current_Manifest-10_0_100_Transport
-    -->
-    <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest100100TransportVersion)</MicrosoftNETRuntimeEmscriptenVersion>
-    <!-- workloads -->
+
+    <!-- emscripten workload package when testing workloads -->
+    <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftDotNetApiCompatTaskPackageVersion)</MicrosoftNETRuntimeEmscriptenVersion>
```

Comment:
Note that reading from another property in Versions.props is an anti-pattern as this will have the wrong value when building from source. That overrides versions after the Versions.props import. We usually put things that read from Version.Details.props / Versions.props into Directory.Build.props.

Might not matter here if this path isn't exercised in source-build but wanted to mention it.

See https://github.com/dotnet/sdk/blob/0d31ba892465268a030b3563df0bddef39855f6e/Directory.Build.props#L97 for an example

Reply:
Actually, this doesn't just apply to source-build but also to the repo build inside the VMR. So it might matter here. Not sure.

Reply:
To clarify - if it's shipped out of source build. My understanding may be stale, but I didn't think we did that for wasm.

Reply:
https://github.com/dotnet/runtime/issues/122354